packages/block-editor/src/components/off-canvas-editor/link-ui.js

```diff
@@ -3,6 +3,7 @@
 /**
  * WordPress dependencies
  */
+import { __unstableStripHTML as stripHTML } from '@wordpress/dom';
```

Comment:
@scruffian I don't understand why but here I _didn't_ need to use the unstable export. But in the Nav Link block I _did_ else it would error.

Any ideas!?